server/config/express.js

```diff
@@ -17,17 +16,63 @@ const express = require('express'),
 	xml2js = require('xml2js').parseString,
 	axios = require('axios'),
 	authMiddleware = require('../src/middlewares/auth'),
-	{ errors, isCelebrate } = require('celebrate');
+	{ errors, isCelebrate } = require('celebrate'),
+	bunyan = require('bunyan'),
+	seq = require('bunyan-seq'),
+	bunyanMiddleware = require('bunyan-middleware');
 
 const swaggerSpec = swaggerJSDoc({
 	swaggerDefinition: swaggerDefinition,
 	apis: ['./docs/**/*.yaml'],
 });
 
+function wrappedStdout() {
+	return {
+		write: (entry) => {
+			// Format console output to be human readable
+			var logObject = JSON.parse(entry);
```

Comment:
Var is pretty bad to use, use `let` or `const`, in this case it's `const` as `logObject` never gets re-assigned to.
```suggestion
			const logObject = JSON.parse(entry);
```